backend/leaderboard/views.py

```
@@ -740,6 +742,22 @@ def community(self, request):
         user_rank = None
         user_total_points = None
 
+        def community_rank(user):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `community_rank`

(ANN202)